common/addressing/ip.go

```diff
@@ -36,7 +36,7 @@ type CiliumIP interface {
 
 type CiliumIPv6 []byte
 
-// - Be an IPv6 address
+// NewCiliumIPv6 returns a IPv6 address or nil.
```

Comment:
The previous sentence was regarding the requirements of `address` so you can change to:
```
NewCiliumIPv6 returns a CiliumIPv6 if the given `address` is:
- An IPv6 address
```